docs/services/manage-data/data-service/schemas.md

```diff
@@ -615,15 +617,14 @@ await sdk.data.transitions.create(newSchema.id, {
 ```
 {% endtab %}
 
-{% tab title="Second Tab" %}
-
-{% endtab %}
 {% endtabs %}
 
 ## Indexes
 
 The Index object is identified by an id and a name. An index is set on a specific property in a Schema. This property is defined in the Fields object by the name and type attribute. The index is tailored with the following attributes:
 
+<!-- Some attributes are not described -->
+
```

Comment:
The background and system attributes are not described properly. 

To be discarded once addressed.